prow/scripts/reserve-ip-address.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+set -o errexit
+
+if [ -z "$GCLOUD_PROJECT_NAME" ]; then
```

Comment:
Shouldn't we make the variable names consistent in all the scripts? Here we have GCLOUD_PROJECT_NAME and in the `provision-cluster.sh` we have PROJECT. Both those variables handles the same parameter.